apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep2.vue

```diff
@@ -1,8 +1,276 @@
+                      @click="handleClickNextButton"
+            >
+                <!--                song-lang-->
+                {{ $t('Continue') }}
+            </p-button>
+        </div>
+        <!--        song-lang-->
+        <delete-modal :header-title="$t('Are you sure you want to start from selecting plugin?')"
+                      :visible.sync="state.deleteModalVisible"
+                      :contents="$t('You cannot undo this action.')"
+                      @confirm="handleClose"
+        />
+    </div>
 </template>
 
 <script lang="ts" setup>
-(() => {
+import { computed, reactive } from 'vue';
+
+import {
+    PFieldTitle, PFieldGroup, PTextInput, PToggleButton, PSelectDropdown, PButton, PTextButton,
+} from '@spaceone/design-system';
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+
+import { i18n } from '@/translations';
+
+import TagsInputGroup from '@/common/components/forms/tags-input-group/TagsInputGroup.vue';
+import type { Tag } from '@/common/components/forms/tags-input-group/type';
+import DeleteModal from '@/common/components/modals/DeleteModal.vue';
+import ErrorHandler from '@/common/composables/error/errorHandler';
+import { useFormValidator } from '@/common/composables/form-validator';
+
+import CollectPluginContents
+    from '@/services/asset-inventory/collector/modules/CollectPluginContents.vue';
+import { useCollectorFormStore } from '@/services/asset-inventory/store/collector-form-store';
+
+
+const emit = defineEmits([
+    'update:currentStep',
+]);
+
+const collectorFormStore = useCollectorFormStore();
+const collectorFormState = collectorFormStore.$state;
+
+const state = reactive({
+    loading: true,
+    plugin: {},
+    pluginId: computed(() => collectorFormState.selectedCollector.plugin_id),
+    tags: {},
+    collectorNames: [] as any[], // FIXME: type
+    versions: [] as any[], // FIXME: type
+    isAutoUpgrade: false,
+    isTagsValid: true,
+    deleteModalVisible: false,
+});
+
+const {
+    forms: {
+        name,
+        version,
+    },
+    setForm,
+    invalidState,
+    invalidTexts,
+    isAllValid,
+} = useFormValidator({
+    name: '',
+    version: '',
+}, {
+    name(value: string) {
+        if (value.length < 2) {
+            return i18n.t('PLUGIN.COLLECTOR.CREATE.NAME_INVALID_MIN');
+        } if (state.collectorNames.includes(value)) {
+            return i18n.t('PLUGIN.COLLECTOR.CREATE.NAME_INVALID_DUPLICATED');
+        }
+        return '';
+    },
+    version(value: string) {
+        if (value.length === 0) {
+            return i18n.t('PLUGIN.COLLECTOR.CREATE.VERSION_INVALID_REQUIRED');
+        }
+        return '';
+    },
+});
+
+const getNames = async () => {
+    // TODO: You need to check if there are any API changes.
+    const res = await SpaceConnector.client.inventory.collector.list({
+        query: {
+            only: ['name'],
+        },
+    });
+    state.collectorNames = res.results.map((v) => v.name);
+};
```

Comment:
suggestion: The name of function is 'get'Names, so I think it should return names. It seems to getAndSetNames :)
Anyway, where's try/catch?🫢